docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,62 @@
+# Programming Calico Policy in Kubernetes
+The [Calico-Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) will allow you to program Networking policy directly from your Kubernetes pod config. 
+
+
+### The Basics
+The Calico Networking plugin will look for a `policy` key in the annotations section of your pod's metadata. This `policy` key should map to a single string that outlines networking behavior semantically.
+
+Programming policy is as simple as saying what you want your pod to do!
```

Comment:
As fun as this line is,  I think we should kill it. :(

Yep.  I'm a fun killer...